src/Game.js

```diff
@@ -0,0 +1,51 @@
+import Player from '../src/Player';
+import Round from '../src/Round';
+
+class Game {
+  constructor(data) {
+    //   constructor(response_code, categories, clues ) {
+    // Object.assign(this, { response_code, categories, clues });
+    this.categories = data.categories
+    this.clues = data.clues;
+    this.players = [];
+    this.roundCounter = 0; // goes up to 3
+    this.fourCategories = [];
+    this.round;
+  }
+
+  getFourCategories() { 
+    let categories = Object.keys(this.categories);
+    let categoriesArray = categories.map((category) => {
+      return {
+        category, id: this.categories[category]
+      }
+    })
+    let randomGenerator = categoriesArray.sort(() => Math.random() - 0.5);
```

Comment:
This looks really clever- can you show me how this works? It looks like it takes in the array and creates a new array in a random order?